templates/signup-role-selection.html

```diff
@@ -0,0 +1,33 @@
+{% extends "auth.html" %}
+
+{% block main %}
+<div class="w-full mx-auto flex flex-col items-center gap-4">
+    <div class="bg-white shadow-md rounded-lg rounded py-4 px-0 md:px-4 lg:px-8">
+        <h2 class="my-0 text-center">{{_('signup_student_or_teacher')}}</h2>
+
+        <div class="mt-8 mx-auto flex flex-wrap lg:flex-nowrap justify-center gap-8">
+            <a href="/signup/student">
+                <div class="w-full border-2 rounded-lg flex flex-col items-center px-8 bg-white text-center border-green-400 bg-gray-200 hover:bg-green-400 cursor-pointer" onclick="window.location = '/tutorial'">
```

Comment:
We have an `<a>` element as well as an `onclick` on the underlining `<div>`? The second also points to an incorrect location. I assume this is from copying it from the landing page code? Would be best to only use the `onclick` as this is consistent with our code base.